atlasdb-config/build.gradle

```diff
@@ -1,32 +1,41 @@
 apply from: "../gradle/publish-jars.gradle"
 apply plugin: 'org.inferred.processors'
+apply plugin: 'com.github.johnrengelman.shadow'
 
 apply from: "../gradle/shared.gradle"
 
+configurations {
+    explicitShadow
+    compile.extendsFrom(explicitShadow)
+    shadow.extendsFrom(explicitShadow)
```

Comment:
Of course - didn't think of that for some reason. Yep, that should be the same (and the timelock artifact produced from that still seems correct)